### ISSUE TITLE
Consider alternative ways to link back to Stanford Libraries website from exhibits.stanford.edu

### ISSUE THREAD

**Issue by caaster** [sul-dlss/exhibits#1361]
A Stanford Libraries staff member has asked that the Stanford Libraries ribbon/logo that appears at the top of every exhibit link back to the Stanford Libraries website. I explained that it was intentional that this is used to link back to the main exhibits.stanford.edu page. However, I said I would ask our UX designer if there was an alternative approach that could be used to satisfy her request. Is there? What is our take on this?

**Comment by ggeisler** [comment]
There is no immediately obvious solution, because we need to point to two different destinations (exhibits home and library website) and currently there is only one UI element (logo) that can be used to point to something outside of the current exhibit.

Three potential solutions I can think of:

- Change the logo link to point to the Library website _instead_ of the exhibits landing page and provide no direct link to the exhibits landing page (which would only make sense if we assume visitors to a given exhibit are only interested in that exhibit; probably often true, but maybe not always).

- Make the logo link _on the exhibits landing page only_ point to the Library website (introduces inconsistency with behavior in individual exhibits, though, and doesn't provide a way to get to the Library website from individual exhibits).

- Change the logo link to point to the Library website (which would make that behavior consistent with other SUL applications) and figure out where to add another affordance in the UI that can be used to point to the exhibits landing page.

The last option above is more effort than the others but probably the best approach. It will need design effort so I'm adding the design needed label to this ticket. If others think one of the first two options might make more sense, though, please chime in.

**Comment by ggeisler** [comment]
(We're solving a SUL problem here so I guess we just do it in Exhibits, although it seems like it could be a useful update for other Spotlight implementers. But we don't know if or what they might want to link the topbar logo to, so maybe it doesn't make sense to do this in Spotlight -- and if even if we wanted to I'm not sure how challenging it would be to make it backwards compatible with existing implementations.)

Recommended solution: Change the link on the SUL logo to `https://library.stanford.edu/` and add a new, initial breadcrumb element that links to exhibits home. 

<img width="1040" alt="Screen Shot 2020-01-29 at 3 37 23 PM" src="https://user-images.githubusercontent.com/101482/73404592-13f8bd80-42af-11ea-882f-11ed0327ea6e.png">

There are a handful of changes related to this:

- [ ] A: Change the link on the SUL logo to `https://library.stanford.edu/`, similar to what we do in Searchworks, PURL, etc.

- [ ] B: Link the exhibit title to the exhibit home page. The link should not change the visual styling of the exhibit title text (just as the SearchWorks heading is linked but doesn't visually change, even when hovered or active).

- [ ] C: Remove the "Home" link from the main menu.

- [ ] D: Add a new, initial breadcrumb item called "Exhibits" that links to `https://exhibits.stanford.edu/`.

- [ ] E: Change the label of the breadcrumb item that currently says "Home" to the title of the exhibit.

- [ ] F: Populate the rest of the breadcrumb as we currently do, with whatever currently follows "Home".

- [ ] Not called out in the mockup above, but we also need to add a breadcrumb, using the same pattern described above, to the exhibit home page, where we don't currently display a breadcrumb. Here's an example exhibit home page after the items above have been implemented:

<img width="1225" alt="Screen Shot 2020-01-29 at 4 35 42 PM" src="https://user-images.githubusercontent.com/101482/73407162-7c4a9d80-42b5-11ea-9f04-79720e2b7988.png">


---

This might come for free with the changes above, but note this pattern applies to the exhibit admin pages as well. For example:

<img width="1180" alt="Screen Shot 2020-01-29 at 4 13 23 PM" src="https://user-images.githubusercontent.com/101482/73406985-14945280-42b5-11ea-9076-f851b8eb813d.png">

---

The biggest drawback I see to the solution proposed above is that exhibits with long titles will now have pretty long breadcrumbs. But it doesn't seem like that is too big of a problem, since the breadcrumb gets the full-width of the content area to display. On narrower viewports that width is more limited, so there will be more line-wrapping,  but exhibits aren't ideal when viewed on mobile viewports in other ways so I'm not sure we're making things much worse.

An example of how the solution would look for an exhibit with a long title:

<img width="1040" alt="Screen Shot 2020-01-29 at 3 41 50 PM" src="https://user-images.githubusercontent.com/101482/73405610-91bdc880-42b1-11ea-8bb4-d44a71a31985.png">

**Comment by anarchivist** [comment]
Bumping this for discussion by the service team, just because there's a few LIBWEBF tickets about this and it came up in the OEG meeting today (2021-05-19): [LIBWEBF-24456](https://jirasul.stanford.edu/jira/browse/LIBWEBF-24456), [LIBWEBF-23979](https://jirasul.stanford.edu/jira/browse/LIBWEBF-23979).

**Comment by anarchivist** [comment]
Also, worth noting: the header on SearchWorks links to `https://library.stanford.edu/` too.

**Comment by ggeisler** [comment]
@anarchivist I've seen the exhibit feedback comments included in the JIRA before and [proposed a solution in January of last year](https://github.com/sul-dlss/exhibits/issues/1361#issuecomment-580015103). As far as I know, nothing relevant has changed and that is still my proposal, and I believe is one that would address the feedback that has come in (and that you presumably heard about in OEG).

In other words, I believe the work just needs prioritizing (it's probably **not** a trivial amount of work), not more discussion. We should ask @caaster to okay it from a service manager perspective but I don't think there is any need for further discussion among the service team, unless Cathy has some concerns I'm not anticipating.

**Comment by caaster** [comment]
@anarchivist I agree with everything @ggeisler has outlined above

**Comment by anarchivist** [comment]
Ah, thanks for clarifying, both. I can follow up with @caaster.
